### DISCUSSION TITLE
Change the name from esphome_samsung_ac to esphome_samsung_hvac_bus

### DISCUSSION THREAD

**Discussion by lanwin** [omerfaruk-aran/esphome_samsung_hvac_bus#196] [Announcements]
Hi, this component changed a bit its targets over the time. To better reflect that, I changed the name of the project.

I would also point out that this component focuses on controlling the devices via bus. Not via REST and not via Smartthings.

**Comment by omerfaruk-aran** [comment]
Hello, I think there was a typo, it should be 'hvac', not 'havc'. Did you write that way on purpose?

**Reply by lanwin** [reply]
Ah thank you. Did noticed that the chars are got swapped.
